conf/mini_provisioner/v2/support.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+support_bundle:
+    - cortx support_bundle create

```

Comment:
Wondering whether "cortx" is the command ? If so will it create support bundle for HA ?
In general every component has a script name having the component name it. For example, below is from S3:
support_bundle:
  - sh /opt/seagate/cortx/s3/scripts/s3_bundle_generate.sh

Reply:
The "cortx" is a CLI which is owned by HA and installed as part of rpm. Currently, generating support bundle for HA is done using the cortx cli, by passing above arguments to that cli.

Reply:
Thanks for the explanation